quartz/components/ExplorerNode.tsx

```diff
@@ -29,19 +29,25 @@ export type FolderState = {
 export class FileNode {
   children: FileNode[]
   name: string
+  displayName: string
```

Comment:
good idea should prob also update examples in docs to work by modifying `displayName` over `name`